framework-support/jobrunr-spring-boot-3-starter/build.gradle

```
@@ -7,14 +7,11 @@ plugins {
 }
 
 compileJava {
-    sourceCompatibility = JavaVersion.VERSION_17
-    targetCompatibility = JavaVersion.VERSION_17
     options.release = 17
 }
 
 compileTestJava {
-    sourceCompatibility = JavaVersion.VERSION_17
-    targetCompatibility = JavaVersion.VERSION_17
+    options.release = 21
```

Should we stick to 17?

```suggestion
    options.release = 17
```